src/legacy/core_plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -345,6 +352,9 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
       this.handler.destroy();
       this.handler.getElement().remove();
     }
+    if (this.autoRefreshFetchSubscription) {
+      this.autoRefreshFetchSubscription.unsubscribe();
+    }
```

Comment:
Question: since `this.autoRefreshFetchSubscription` is set in the constructor, it can't be undefined? Is there a reason for 
```
private autoRefreshFetchSubscription?: Subscription; 
```

Reply:
Very true, fixed.